app/controllers/api/v1/lookup_keys_controller.rb

```diff
@@ -11,9 +11,10 @@ class LookupKeysController < V1::BaseController
       param :per_page, String, :desc => "number of entries per request"
 
       def index
-        @lookup_keys = LookupKey.
-          authorized(:view_external_variables).
-          search_for(*search_options).paginate(paginate_options)
+        @lookup_keys = PuppetclassLookupKey.authorized(:view_external_variables).
```

Comment:
we need both types, searching the base class didn't work because we moved scoped search out of the base class since there was a problem with scoped search that the first class it sees it uses only that.
As a result defining some things to search in the base class and others in `VariableLookupKey` causes an error when searching on what was defined in the child class